source/includes/connection-snippets/x509-snippets/php-connection.php

```diff
@@ -0,0 +1,12 @@
+<?php
+/* Begin x509 connection */
+$subject = "mongodb+srv://<your_subject>"; 
+$manager = new MongoDB\Driver\Manager(
+    $subject + "@<cluster-url>/?ssl=true&authMechanism=MONGODB-X509",
+  [],
+  [
+    "pem_file" => "/etc/certs/mongodb/v3/client.pem",
+    "ca_file" => "/etc/certs/mongodb/v3/ca.pem",
+  ]
+);
+/* End x509 connection */
```

Comment:
I propose the following source for this file:

```php
/* Begin x509 connection */
$client = new MongoDB\Client(
    'mongodb+srv://<your_subject>@<cluster-url>/?authMechanism=MONGODB-X509',
    [
        'tlsCAfile' => '/etc/certs/mongodb/v3/ca.pem',
        'tlsCertificateKeyFile' => '/etc/certs/mongodb/v3/client.pem',
    ]
);
/* End x509 connection */
```

Note the following:

 * I replaced usage of the extension (i.e. MongoDB\Driver\Manager) with the library (i.e. MongoDB\Client). This is consistent with what you were doing in the SCRAM example. For all intents and purposes, the constructors take and support the same arguments. The library's MongoDB\Client class passes its constructor arguments directly to the MongoDB\Driver\Manager class, which is creates internally. The MongoDB\Client supports some extra options in the third argument, but we're not using it in any of these examples and it's not relevant to auth at all.
 * I replaced the deprecated driver options (third constructor argument) with [unified URI options](https://github.com/mongodb/specifications/blob/master/source/uri-options/uri-options.rst) in the second argument, which I also mentioned in some other comments for other language examples. Since we're now using URI options, it's certainly possible to stuff _everything_ into a single connection string (i.e. first argument), but I broke it out in the interest of readability. When constructing a client, the PHP driver basically merges any options in the second constructor parameter on top of the connection string (i.e. second parameter options take precedence). The third parameter is used for driver options and is handled entirely separately (different keys and rules for merging), but it's no longer relevant for any of these examples if we use can use unified URI options. 
 * I removed `ssl=true` from the URI, since the current version of the driver implicitly enables TLS when one or more TLS-related options are specified (we're using two here). For an explanation of this, you can see the [extension's changlog](https://www.php.net/manual/en/mongodb-driver-manager.construct.php#refsect1-mongodb-driver-manager.construct-changelog).